static/app/views/settings/dynamicSampling/index.tsx

```diff
@@ -22,10 +26,43 @@ export default function DynamicSamplingSettings() {
   const hasWriteAccess = useHasDynamicSamplingWriteAccess();
   const hasReadAccess = useHasDynamicSamplingReadAccess();
 
+  if (
+    hasDynamicSamplingFeature(organization) &&
+    !hasDynamicSamplingCustomFeature(organization)
+  ) {
+    return (
+      <Alert.Container>
```

Comment:
we probably do not need the container, since there is no element below the alert